bool exist_enemy_flag
bool exist_pad_flag
bool exist_friend_flag
int32 enemy_num
int32 pad_num
int32 friend_num
float64[5] enemy_ymin
float64[5] enemy_xmin
float64[5] enemy_ymax
float64[5] enemy_xmax
float64[5] pad_ymin
float64[5] pad_xmin
float64[5] pad_ymax
float64[5] pad_xmax
float64[5] friend_ymin
float64[5] friend_xmin
float64[5] friend_ymax
float64[5] friend_xmax